mst/restapi.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using REST API in Managed Service for TimescaleDB
+excerpt: Use REST API in Managed Service for TimescaleDB for integration and automation
+product: mst
+keywords: [rest api]
+---
+
+# Using REST API in Managed Service for TimescaleDB
+
+Managed Service for TimescaleDB has an API for integration and automation tasks.
+For information about using the endpoints, see the [API Documentation](https://api.aiven.io/doc/).
+MST offers an HTTP API with token authentication and JSON-formatted data. You
+can use the API for all the tasks that can be performed using the web console.
+
+<procedure>
```

Comment:
This probably needs to be laid out as a process/procedure.

Reply:
I added s sentence now that explains the need to create an authentication token.